types/jquery.pnotify/index.d.ts

```diff
@@ -192,9 +191,10 @@ interface PNotifyOptions {
      */
     icon?: any;
     /**
-     * The animation to use when displaying and hiding the notice. "none", "show", "fade", and "slide" are built in to jQuery. Others require jQuery UI. Use an object with effect_in and effect_out to use different effects.
+     * The animation to use when displaying and hiding the notice. "none", "show", "fade", and "slide" are built in to jQuery.
+     * Others require jQuery UI. Use an object with effect_in and effect_out to use different effects.
      */
-    animation?: any;
+    animation?: boolean | string;
```

Comment:
Why boolean? I can't seem to find any information in the source where it is ever set to false (other than in the PNotifyStack, which isn't necessarily the same thing)

Reply:
agreed should be string only